action_server/src/action_server/actions/action.py

```diff
@@ -107,16 +107,19 @@ def _configure(self, robot, config):
         raise NotImplementedError
 
     def start(self):
-        rospy.loginfo("Starting executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Starting executing of action {}.".format(self.get_name()))
         self._start()
         return self._execute_result
 
     def _start(self):
         raise NotImplementedError
 
     def cancel(self):
-        rospy.loginfo("Canceling executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Canceling executing of action {}.".format(self.get_name()))
         return self._cancel()
 
     def _cancel(self):
         raise NotImplementedError
+
```

Comment:
This is the same  change as in https://github.com/tue-robotics/action_server/pull/80? That's certainly going to be a (trivial) merge conflict

Reply:
It is the same commit, which is already on master. Should have rebased this after merging the other one